conformance/results/mypy/dataclasses_final.toml

```diff
@@ -0,0 +1,22 @@
+conformant = "Fail"
```

Comment:
This should be either "Unsupported" or "Partial". The summary report generator won't know what to do with "Fail".

Reply:
I guess on second thought maybe I'll go with Partial for mypy, too. It does support the core functionality of defining a final dataclass field and preventing reassignment of it, although forcing a default value is kind of a major limitation.